package/nginx-http.conf

```diff
@@ -5,11 +5,23 @@ server {
     error_log   /var/log/nginx/rmt_http_error.log;
     root        /usr/share/rmt/public;
 
+    include /etc/nginx/rmt-auth*.d/auth-location*.conf;
+
     location / {
         autoindex off;
     }
 
     location /repo {
         autoindex on;
+        include /etc/nginx/rmt-auth*.d/auth-handler*.conf;
+    }
+
+    location = /repo/repoindex.xml {
+        return 301 https://$host$request_uri;
+    }
+
+    # An alias to RMT CA certificate, so that it can be downloaded to client machines.
+    location /rmt.crt {
```

Comment:
Do we really need this part?

Reply:
Robert has asked to make the certificate available over HTTP as well (to avoid handling SSL exceptions on the client side -- `rmt-client-setup` isn't used in the cloud).

Reply:
@digitaltom and I talked and this should be fine, as the supported setup is through https only.

Reply:
OK, cool. Thanks.